kv.go

```diff
@@ -629,6 +630,14 @@ func (kv *kvs) Create(key string, value []byte) (revision uint64, err error) {
 		return kv.Update(key, value, e.Revision())
 	}
 
+	// Check if the expected last subject sequence is not zero which implies
+	// the key already exists.
+	if aerr, ok := err.(*APIError); ok {
+		if aerr.ErrorCode == 10071 {
```

Comment:
Let me know if there is a better way to identify this particular error type.

Reply:
maybe add it as one of the constants here: https://github.com/nats-io/nats.go/blob/main/jserrors.go#L112
so that it represents this error from the server: 
https://github.com/nats-io/nats-server/blob/e42260cc80f793006ef7f90cc91ede26fe14073f/server/jetstream_errors_generated.go#L536